src/NHibernate.Test/Async/Linq/NullComparisonTests.cs

```diff
@@ -260,13 +265,7 @@ public async Task NullEqualityWithNotNullAsync()
 			q = session.Query<AnotherEntityRequired>().Where(o => (o.NullableOutput + o.Output) == o.Output);
 			await (ExpectAsync(q, Does.Not.Contain("is null").IgnoreCase));
 
-			q = session.Query<AnotherEntityRequired>().Where(o => (o.Input + o.Output) == o.Output);
```

Comment:
Changed to `o.Output + o.Output` as otherwise it fails in oracle as in oracle concatenating `null` with a non empty string will not result in a `null` value like in the other databases. I think that we should add an additional logic when concatenating strings in order to have a consistent behaviour across various databases and possible align with Linq-to-Object so that `null + "test"` will not result in a `null` value.